src/diet.ts

```diff
@@ -350,12 +297,10 @@ export function runDiet(): void {
   }
 
   if (globalOptions.ascending) useIfUnused($item`borrowed time`, "_borrowedTimeUsed", 5 * MPA);
+}
 
-  fillSomeSpleen();
-  fillStomach();
-  fillLiver();
-
-  if (!get("_distentionPillUsed") && 1 <= myInebriety()) {
+function pillCheck(): void {
```

Comment:
is it worth adding the "just use a map if you have them" thing to this PR, or should we add that just after?

Reply:
after